src/test/java/edu/hm/hafner/analysis/parser/AntJavacParserTest.java

```diff
@@ -228,7 +228,15 @@ protected AntJavacParser createParser() {
     @Test
     void shouldParseJavaWarningsInMavenCompilerPlugin() {
         Report warnings = parse("issue63346.log");
-        assertThat(warnings).hasSize(0);
+        assertThat(warnings).hasSize(2);
```

Comment:
Why did you change that back? This parser is broken and should not report a warning.